sparkler-plugins/fetcher-htmlunit/src/test/java/edu/usc/irds/sparkler/plugin/HtmlUnitFetcherTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.usc.irds.sparkler.plugin;
+
+import edu.usc.irds.sparkler.model.FetchedData;
+import edu.usc.irds.sparkler.model.Resource;
+import edu.usc.irds.sparkler.util.TestUtils;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class HtmlUnitFetcherTest {
+
+
+	@Test
+	public void testJBrowserImage() throws Exception {
```

Comment:
why not test against an actual page where the HTML content != what you would get from a HTTP GET request?